e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -2249,6 +2257,104 @@ describeEE("scenarios > dashboard > dashboard cards > click behavior", () => {
     });
   });
 
+  /**
+   * In https://github.com/metabase/metabase/issues/46519 we added { "stage-number": number }
+   * to target dimension. This test ensures that existing dashboards without this information
+   * still work the same as they did before that epic.
+   */
+  describe("click behavior parameter mappings for questions without target stage index", () => {
```

Comment:
This test is basically the same as repro for #16334.
The difference is that #16334 repro will now have `stage-number` attribute present, while this test won't.
This way we don't have to couple the assertions.